db/view/view_update_generator.cc

```diff
@@ -179,6 +179,8 @@ future<> view_update_generator::start() {
                     // Need to add sstables back to the set so we can retry later. By now it may
                     // have had other updates.
                     std::move(sstables.begin(), sstables.end(), std::back_inserter(_sstables_with_tables[t]));
+                    // Sleep a bit, to avoid a tight loop repeatedly spamming the log with the same message.
+                    seastar::sleep(std::chrono::seconds(1)).get();
```

Comment:
Maybe not in this PR, but we should make this exponential back-off, with a ceiling of say 1 minute.

Reply:
I don't know if it's important. Basically, if the generator fails after a long time (e.g., after working for 10 minutes), it doesn't matter if we sleep for 1 second or 60 seconds. If it fails after a long time (in this case, it failed after a fraction of a millisecond), again it doesn't matter if we sleep for 1 or 60 seconds - both are negligible performance-wise, and in both the log spam as at a manageable level.